modules/network/src/integrationTest/java/org/apache/ignite/internal/network/netty/ItConnectionManagerTest.java

```diff
@@ -308,7 +305,26 @@ public void testConnectMisconfiguredClient() throws Exception {
     public void testStartTwice() throws Exception {
         ConnectionManagerWrapper server = startManager(4000);
 
-        assertThrows(IgniteInternalException.class, server.connectionManager::start);
+        IgniteInternalException exception = (IgniteInternalException) assertThrows(
+                IgniteInternalException.class,
+                server.connectionManager::start,
+                "Attempted to start an already started connection manager"
+        );
+
+        assertEquals("IGN-CMN--1", exception.codeAsString());
```

Comment:
Is this how we render 65535? Let's file an issue and fix this separately

Reply:
https://issues.apache.org/jira/browse/IGNITE-24615